src/main/java/com/redhat/developers/msa/hola/HolaResource.java

```diff
@@ -132,24 +131,34 @@ public String health() {
      * @return The feign pointing to the service URL and with Hystrix fallback.
      */
     private AlohaService getNextService() {
-        final String serviceName = "aloha";
-        // This stores the Original/Parent ServerSpan from ZiPkin.
-        final ServerSpan serverSpan = brave.serverSpanThreadBinder().getCurrentServerSpan();
-        final CloseableHttpClient httpclient =
-            HttpClients.custom()
-                .addInterceptorFirst(new BraveHttpRequestInterceptor(brave.clientRequestInterceptor(), new DefaultSpanNameProvider()))
-                .addInterceptorFirst(new BraveHttpResponseInterceptor(brave.clientResponseInterceptor()))
-                .build();
-        String url = String.format("http://%s:8080/", serviceName);
+        String url = System.getenv("ALOHA_SERVER_URL");
+        if (null == url || url.isEmpty()) {
+            String host = System.getenv("ALOHA_SERVICE_HOST");
+            String port = System.getenv("ALOHA_SERVICE_PORT");
+            if (null == host) {
+                url = "http://aloha:8080/";
+            } else {
+                url = String.format("http://%s:%s", host, port);
+            }
+        }
+
+        SpanContext spanContext = (SpanContext) servletRequest.getAttribute("tracing.requestSpan");
+
+        final CloseableHttpClient httpclient;
+        if (tracer != null) {
+            Span span = tracer.buildSpan("GET").asChildOf(spanContext).start();
```

Comment:
HTTP verb is fine(generic) but we can make it look better :). 

Some guidelines on operation names: 
https://github.com/opentracing/specification/blob/master/specification.md#start-a-new-span